index.src.html

```diff
@@ -0,0 +1,797 @@
+  1. Let |intermediate memory measurement| be implementation-defined memory measurement of |agent clusters|.
+  1. Let |memory measurement| be the result of [=creating a new memory measurement=] given |intermediate memory measurement|.
+  1. Queue a task that resolves |promise| with |memory measurement|.
+</div>
+
+
+Converting an intermediate memory measurement to the result {#converting-an-intermediate-memory-measurement-to-the-result}
+--------------------------------------------------------------------------------------------------------------------------
+<div algorithm>
+  To <dfn lt="creating a new memory measurement|create a new memory measurement">create a new memory measurement</dfn>
+  given an [=intermediate memory measurement=]  |intermediate measurement|:
+  1. Let |bytes| be 0.
+  1. [=For each=] [=intermediate memory breakdown entry=] |intermediate entry| in |intermediate measurement|:
+      1. Set |bytes| to |bytes| plus |intermediate entry|'s [=intermediate memory breakdown entry/bytes=].
+  1. Let |breakdown| be a new [=list=].
+  1. [=set/For each=] [=intermediate memory breakdown entry=] |intermediate entry| in |intermediate measurement|:
+      1. Let |breakdown entry| be the result of [=creating a new memory breakdown entry=] given |intermediate entry|.
+      1. [=list/Append=] |breakdown entry| to |breakdown|.
+  1. Return a new {{MemoryMeasurement}} whose:
+      - {{MemoryMeasurement/bytes}} is |bytes|,
+      - {{MemoryMeasurement/breakdown}} is |breakdown|.
+</div>
+
+<div algorithm>
+  To <dfn lt="creating a new memory breakdown entry|create a new memory breakdown entry">create a new memory breakdown entry</dfn>
+  given an [=intermediate memory breakdown entry=]  |intermediate entry|:
+  1. Let |attribution| a new [=list=].
+  1. [=set/For each=] {{JavaScript realm}} |realm| in |intermediate entry|'s [=intermediate memory breakdown entry/realms=]:
+      1. Let |attribution entry| be the result of [=creating a new memory attribution=] given |realm|.
+      1. [=list/Append=] |attribution entry| to |attribution|.
+  1. Return a new {{MemoryBreakdownEntry}} whose:
+      - {{MemoryBreakdownEntry/bytes}} is |intermediate entry|'s [=intermediate memory breakdown entry/bytes=],
+      - {{MemoryBreakdownEntry/attribution}} is |attribution|,
+      - {{MemoryBreakdownEntry/userAgentSpecificTypes}} is |intermediate entry|'s [=intermediate memory breakdown entry/user agent specific types=].
+</div>
+
+<div algorithm>
+  To <dfn lt="creating a new memory attribution|create a new memory attribution">create a new memory attribution</dfn>
+  given a {{JavaScript realm}} |realm|:
+  1. Let |token| be |realm|'s [=global object=]'s [=WindowOrWorkerGlobalScope/memory attribution token=].
+  1. If |token|'s [=memory attribution token/cross-origin aggregated flag=] is true, then
+      1. Return a new {{MemoryAttribution}} whose:
+          - {{MemoryAttribution/url}} is "cross-origin-url",
```

Comment:
Very much a nit:

```suggestion
          - {{MemoryAttribution/url}} is "`cross-origin-url`",
```

and similar for other strings.

Reply:
Done.